packages/app/src/api/query.api.ts

```diff
@@ -0,0 +1,17 @@
+import { BASE_API_URL, useBetterQuery } from '.'
+import { apiRequest } from '../utils/request'
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function useGetLocalSqlQuery<T = any>(query: string) {
+  const queryFn = () =>
+    apiRequest({
+      url: `${BASE_API_URL}/localdb/query`,
+      method: 'POST',
+      body: { query },
+      credentials: 'include',
+    })
+  return useBetterQuery<T[], Error>({
+    queryKey: ['localdb/query'],
```

Comment:
You'll want to make this key unique per query string passed in. As is it'll reuse the same key for any query string passed in